src/js/media/html5.js

```diff
@@ -438,8 +415,6 @@ vjs.Html5.prototype.addRemoteTextTrack = function(options) {
     if (track.readyState >= 2) {
```

Comment:
If we're going for default behavior now would we not just kill this whole function, or do we still need to force metadata tracks to hidden?